### DISCUSSION TITLE
Application is not working after upgrading to webpack 5

### DISCUSSION THREAD

**Discussion by vbothe** [webpack/webpack#15033] [Q&A]
# Bug report

**What is the current behavior?**
Application is not running after upgrading to wepack 5. I am getting below error when try to load application
Uncaught Error: [$injector:modulerr] Failed to instantiate module itest due to:
Error: [$injector:modulerr] Failed to instantiate module core due to:
Error: [$injector:unpr] Unknown provider: e

I have compared webpack 4 and webpack 5 bundle on my system and could see in webpack 5 bundle, runtime.js file is not getting created.

I am using angular 1.8.2

**If the current behavior is a bug, please provide the steps to reproduce.**
After build creation application is not working and getting above error in console with empty page.

**What is the expected behavior?**

**Other relevant information:**
webpack version:  5.37.0
Node.js version: 12.14.0
Operating System: 
Additional tools:

**Comment by alexander-akait** [comment]
Please do not ignore `If the current behavior is a bug, please provide the steps to reproduce.`, if you need runtime, you can enable it using `optimization.runtime`

**Reply by vbothe** [reply]
I am using below optimization details
    `    optimization: {
      moduleIds: "deterministic",
      minimizer: [
        new TerserPlugin({
          terserOptions: {
            format: {
              comments: false
            }
          },
          extractComments: false
        }),
        new CssMinimizerPlugin({
          minimizerOptions: {
            preset: [
              "default",
              {
                discardComments: { removeAll: true }
              }
            ]
          }
        })
      ],
      runtimeChunk: { name: "single" },
      splitChunks: {
        chunks: "all",
        maxInitialRequests: 4,
        maxSize: 4000000,
        minSize: 400000,
        cacheGroups: { vendor: { test: /[\\/]node_modules[\\/]/ } }
      }
    }`

**Reply by alexander-akait** [reply]
Based on your report the problem in angular
